src/solvers/dgsem_tree/indicators.jl

```diff
@@ -151,6 +151,41 @@ const IndicatorLoehner = IndicatorLöhner
 end
 
 
+"""
+    IndicatorIDP
+
+Indicator used for subcell shock-capturing used by Rueda-Ramírez, Pazner, Gassner.
+"""
+struct IndicatorIDP{Variable, Cache} <: AbstractIndicator
+  # alpha_max::RealT
+  # alpha_min::RealT
+  # alpha_smooth::Bool
```

Comment:
```suggestion
  # TODO
  # alpha_max::RealT
  # alpha_min::RealT
  # alpha_smooth::Bool
```
Needs to be fixed later